xbmc/guilib/guiinfo/MusicGUIInfo.cpp

```diff
@@ -244,9 +248,13 @@ bool CMusicGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
         break;
       }
       case PLAYER_DURATION:
-        if (!g_application.GetAppPlayer().IsPlayingAudio())
+      {
+        const auto& components = CServiceBroker::GetAppComponents();
+        const auto appPlayer = components.GetComponent<CApplicationPlayer>();
+        if (!appPlayer->IsPlayingAudio())
           break;
         // fall-thru is intended.
+      }
         [[fallthrough]];
```

Comment:
Unrelated, but while we are at it, the comment could be removed, now that we have `[[fallthrough]]`

Reply:
removed.